### PR TITLE
Login/sign up/reset password pages

### PR DESCRIPTION
**Pull request by megan-godfrey**
Improves the layout of the account login, sign up, reset password pages

In the process, adds generic css for cards and headers

Also tidies up `application.css` by moving feature specific css to different files

Note: styling of submit buttons has not been updated as Marco will be making them fancy in future

image

### CODE REVIEW

**Review by Cena-Studio — CHANGES_REQUESTED** [review]
Basically LGTM. A small improvement could be made before it is approved as commented below.

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -73,73 +67,45 @@ body {
...
+    margin-top: var(--navbar-height);
 }
 
-#nav-bar .btn:not(:hover) {
-    color: #586674;
+h1, h2 {
+    font-weight: 800;
+    background-color: var(--color-main);
+    background: var(--gradient-blue-green);
+    -webkit-background-clip: text;
```
[the remark this answers is not in the document]
Comment (megan-godfrey) [reply app/assets/stylesheets/application.css:82]: Happy to hear if there's a better way - I was also not confident on compatibility, hence why I included the `background-color` as a back up

[the remark this answers is not in the document]
Comment (Cena-Studio) [reply app/assets/stylesheets/application.css]: FYI a more stable but also much more complicated approach is to add an svg mask to the page and link the mask as the filter of the element. We could keep the current implementation until any bug occurs.

**Review by Cena-Studio — APPROVED** [review]
LGTM!

**Comment by Womblue** [comment]
Looks like the only issue I had with the branch is Turbo-related, so this all LGTM!

**Pull request merged**